src/scss/03-widgets/_list-item.scss

```diff
@@ -87,3 +87,31 @@
 		}
 	}
 }
+
+// OS HighContrast Enabled -------------------------------------------------------
+///
+.os-hight-contrast {
+	&.desktop,
+	&.tablet.landscape {
+		.list {
+			.list-item:hover,
+			.list-item-selected {
+				position: relative;
+
+				&:before {
+					border: 2px solid var(--color-focus-outer);
+					border-right-width: 0;
+					border-left-width: 5px;
```

Comment:
border-width of value 5px doesn't exist on OSUI DS, couldn't this be 3px?

Reply:
Done!